### ISSUE TITLE
Saving a report doesn't work when passing two catalogs into the CLI

### ISSUE THREAD

**Issue by natemccurdy** [camptocamp/puppet-catalog-diff#37]
When using the `--output_report` flag and passing in two catalogs, a report doesn't get saved.

For example, this does not write a `report.json`:
```
puppet catalog diff catalog-a.json catalog-b.json --output_report=report.json
```

Looks like the problem is that the code responsible for writing a report only exists in the conditional branch for retrieving a catalog remotely:
https://github.com/camptocamp/puppet-catalog-diff/blob/v2.3.0/lib/puppet/face/catalog/diff.rb#L182-L187

**Comment by natemccurdy** [comment]
Just chiming in to the stale bot doesn't eat this issue. This is still a problem.

**Comment by natemccurdy** [comment]
<insert activity here>

**Comment by raphink** [comment]
Hehe, don't mind the bot, he's not that much of a bad boy (though it could be controlled with some kind of ignore label) 😁

**Comment by raphink** [comment]
Ah, there was an issue open to implement it in the bot https://github.com/probot/stale/issues/110

But the bot closed it 🤣

**Comment by natemccurdy** [comment]
Rather than continuing this fight with a bot, I decided to just fix the problem myself :)
Here's a proposed fix: #50

**Comment by raphink** [comment]
This bit is actually useful 😃
